### PR TITLE
Update areas and owners

### PR DESCRIPTION
**Pull request by ericstj**
/cc @maryamariyan FYI

### CODE REVIEW

File: docs/area-owners.md [hunk 1]
```diff
@@ -48,58 +48,54 @@ Below table shows the combined area owners on this repository:
 | area-Tracing-mono | @lambdageek | |
 | area-Performance-mono | @SamMonoRT | |
 | **System contract assemblies** | | |
-| area-System.AppContext | @safern @Anipik | |
```
Comment (safern) [inline docs/area-owners.md]: Was this removal intentional? Is it going into `System.Runtime`?

Comment (maryamariyan) [reply docs/area-owners.md]: Also, same questions for `area-System.ServiceModel` and `area-System.Runtime.Extensions`.

Comment (ericstj) [reply docs/area-owners.md]: Yes, I merged it into System.Runtime, there is not much value in having a separate area.

Comment (ericstj) [reply docs/area-owners.md]: ServiceModel never lived in our repo and had no relevant issues.  System.Runtime.Extensions isn't a namespace, it was an a assembly with a grab-bag of types which have now all been moved into System.Runtime.

Comment (safern) [reply docs/area-owners.md]: Ok, sounds good. Thanks 😄

**Review by stephentoub — APPROVED** [review]

File: docs/area-owners.md [hunk 2]
```diff
@@ -48,58 +48,54 @@ Below table shows the combined area owners on this repository:
...
-| area-System.Console | @wtgodbe | |
+| area-System.Console | @carlossanlop @eiriktsarpalis | |
 | area-System.Data | @ajcvickers @cheenamalhotra @david-engel | <ul><li>Odbc, OleDb - [@saurabh500](https://github.com/saurabh500)</li></ul> |
 | area-System.Data.SqlClient | @cheenamalhotra @david-engel @karinazhou @JRahnama | Archived component - limited churn/contributions (see https://devblogs.microsoft.com/dotnet/introducing-the-new-microsoftdatasqlclient/) |
-| area-System.Diagnostics | @tommcdon @wtgodbe @krwq | <ul><li>System.Diagnostics.EventLog - [@Anipik](https://github.com/Anipik)</li></ul> |
-| area-System.Diagnostics.Process | @wtgodbe @krwq | |
-| area-System.Diagnostics.Tracing | @noahfalk @tommcdon @Anipik | Packages:<ul><li>System.Diagnostics.DiagnosticSource</li><li>System.Diagnostics.PerformanceCounter - [@Anipik](https://github.com/Anipik)</li><li>System.Diagnostics.Tracing</li><li>System.Diagnostics.TraceSource - [@Anipik](https://github.com/Anipik)</li></ul><br/> |
+| area-System.Diagnostics | @tommcdon @krwq | <ul><li>System.Diagnostics.EventLog - [@Anipik](https://github.com/Anipik)</li></ul> |
+| area-System.Diagnostics.Process | @adsitnik @eiriktsarpalis | |
```
Comment (ahsonkhan) [inline docs/area-owners.md]: Adams GitHub alias is incorrect. Should be @adamsitnik (same elsewhere in this doc).

File: docs/area-owners.md [hunk 3]
```diff
@@ -48,58 +48,54 @@ Below table shows the combined area owners on this repository:
...
 | area-System.Linq.Parallel | @tarekgh @kouvel | |
 | area-System.Management | @Anipik | WMI |
-| area-System.Memory | @ahsonkhan | |
-| area-System.Net | @davidsh @wfurt @scalablecory @eiriktsarpalis @karelz | Included:<ul><li>System.Uri - [@wtgodbe](https://github.com/wtgodbe)</li></ul> |
-| area-System.Net.Http | @davidsh @wfurt @scalablecory @eiriktsarpalis @karelz | |
-| area-System.Net.Security | @davidsh @wfurt @scalablecory @eiriktsarpalis @karelz | |
-| area-System.Net.Sockets | @davidsh @wfurt @scalablecory @eiriktsarpalis @karelz | |
-| area-System.Numerics | @tannergooding @ViktorHofer | |
-| area-System.Numerics.Tensors | @tannergooding | |
+| area-System.Memory | @GrabYourPitchforks @adsitnik | |
```
Comment (ahsonkhan) [inline docs/area-owners.md]:
```suggestion
| area-System.Memory | @GrabYourPitchforks @adamsitnik | |
```

**Comment by ahsonkhan** [comment]
Fixed in https://github.com/dotnet/runtime/pull/32639
